src/Stripe.net/Entities/Invoices/InvoiceTransferData.cs

```diff
@@ -0,0 +1,30 @@
+namespace Stripe
+{
+    using Newtonsoft.Json;
+    using Stripe.Infrastructure;
+
+    public class InvoiceTransferData : StripeEntity
```

Comment:
@ob-stripe Should I have made a common `TransferData` class you think? It's quite verbose. But in Java it's done the same way with internal classes and Go has classes per file

Reply:
Since we don't want to support `transfer_data[amount]` on all resources, just `Charge` I think it's better to duplicate for now.